### ISSUE TITLE
MakeInstall.bat from gibMacOS not showing flashdrive

### ISSUE THREAD

**Issue by farhanpasha** [gnodipac886/MatebookXPro-hackintosh#60]
@ayushyamitabh 
![image](https://user-images.githubusercontent.com/59501091/71763491-50442400-2e91-11ea-836d-1d9940f3154b.png)

**Comment by ayushyamitabh** [comment]
@farhanpasha try formatting your flashdrive - using fat32 might help but shouldn't matter if you use ntfs.

**Comment by ayushyamitabh** [comment]
In case of a corrupted flash drive - on a Windows machine you can do the following, in command prompt 
```sh
> diskpart
> list disk 
# take the drive number from the output
> select disk {number_from_above_output}
> clean
```
Then go to "Disk Management", and reinitialize and format your USB

**Comment by farhanpasha** [comment]
I did format it multiple times. Just with Windows and the way you format it
before using Transmac

**Comment by ayushyamitabh** [comment]
Apologies, if this is a confusion on my end but, are you using Transmac?

**Comment by farhanpasha** [comment]
No. the thing is when I got frustrated with this method and was waiting for Virtual machine to set up, I just googled how to set up usb in windows. And transmac came up. And even there they suggested formatting the data a certain way. I did that and the regular windows partitioning. Then I tried gibmacOS and still no USb showed up so I tried Transmac which idk what it did to my usb but I couldn’t access the USB from windows anymore so I just gave up. But in the end, I ended up making the flash drive using a Virtual Box. Sent from Mail for Windows 10 From: Ayushya AmitabhSent: Monday, January 6, 2020 16:25To: gnodipac886/MatebookXPro-hackintoshCc: Farhan Pasha; MentionSubject: Re: [gnodipac886/MatebookXPro-hackintosh] MakeInstall.bat from gibMacOS not showing flashdrive (#60) Apologies, if this is a confusion on my end but, are you using Transmac?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
